### ISSUE TITLE
Attempting to use -firefox with -chromium installed results in TypeError

### ISSUE THREAD

**Issue by narthollis** [playwright-community/jest-playwright#139]
I have just starting playing with jest + playwright and was trying to run multiple browser configurations when I ran into this issue.

The error I am getting is 
```
 FAIL   browser: firefox  tests/basic.test.ts
  ● Test suite failed to run

    TypeError: Cannot read property 'launch' of undefined

      at getBrowserPerProcess (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:24:41)
      at PlaywrightEnvironment.setup (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:62:41)

 FAIL   browser: firefox  tests/basic.test.js
  ● Test suite failed to run

    TypeError: Cannot read property 'launch' of undefined

      at getBrowserPerProcess (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:24:41)
      at PlaywrightEnvironment.setup (node_modules/jest-playwright-preset/lib/PlaywrightEnvironment.js:62:41)

Test Suites: 2 failed, 2 total
Tests:       0 total
Snapshots:   0 total
Time:        0.339 s
Ran all test suites.
```

jest.config.js
```javascript
module.exports = {
  preset: 'jest-playwright-preset'
};
```

jest-playwright.config.js
```javascript
module.exports = {
  browsers: [
    //  "chromium",
      "firefox"
    ]
};
```

basic.test.js
```javascript
describe('basic test', () => {
    it('should load google.com', async () => {
        await page.goto('https://google.com/');
        expect(await page.title()).toBe("Google");
    });

    it('should also load google.com', async () => {
        await page.goto('https://google.com/');
        expect(await page.title()).toBe("Google");
    })
});
```

**Comment by narthollis** [comment]
Having a read of the source I believe that the issue is that the `checkDependencies` function is always returning the `playwright-chromium` package if it is installed.

https://github.com/playwright-community/jest-playwright/blob/00e15e560087585cc6dc5387e026912c737eb682/src/utils.ts#L20-L29

which is causing `readPackage` to also always return `playwright-chromium` https://github.com/playwright-community/jest-playwright/blob/00e15e560087585cc6dc5387e026912c737eb682/src/utils.ts#L73 

and this flows through to `getPlaywrightInstance` to attempt to find a `firefox` property on the `playwright-chromium` export.
https://github.com/playwright-community/jest-playwright/blob/00e15e560087585cc6dc5387e026912c737eb682/src/utils.ts#L90

**Comment by mmarkelov** [comment]
@narthollis seems like a bug. As I understood you got separated playwright packages.
```js
// package.json
"devDependencies": {
    "playwright-chromium": ">=0.12.1",
    "playwright-firefox": ">=0.12.1",
    "playwright-webkit": ">=0.12.1",
  }
``` 

And you try to running tests for `firefox`?

**Comment by narthollis** [comment]
Yeah, this is correct.

**Comment by narthollis** [comment]
@mmarkelov Any ideas when this might be released?

**Comment by mmarkelov** [comment]
@narthollis sorry for late response. I just published **0.2.4**, that should fix your issue. Thanks!

**Comment by narthollis** [comment]
Thats working! Thanks very much.
